vision.org

```diff
@@ -0,0 +1,32 @@
+#+TITLE: The Future of the ediTDor
+#+AUTHOR: Pedram Hadjian
+
+* Vision
+
+The Web of Things enables us to represent operational digital twins in an interoperable way. The main problem for IoT developers is the access to and understanding of data. Data in the context of IoT can broadly be split into two different categories:
+
+1. Timeseries Data: data coming from sensors, oftentimes called signals, runtime data, samples or others
+
+2. Contextual Data: this type of data answers questions about types of devices, interpretation of timeseries values, topology of assets and locations, taxonomies and more. Sometimes this type of data is called the semantics.
```

Comment:
... or metadata.

Reply:
https://github.com/web-of-things-open-source/proposal/commit/1abdb344aad1f547f36bbcf116c1466007d25a92
- Added "metadata" as alternative to "semantics"